### PR TITLE
Add Python package - Tornado

### PR DESCRIPTION
**Pull request by BKPepe**
Maintainer: me
Compile tested: Turris Omnia, mvebu (cortex-a9), OpenWrt master
Run tested: with example Hello, word - https://pypi.org/project/tornado/

Description:
- Adds new package Tornado (could be in the future dependency for Salt)

### CODE REVIEW

**Comment by BKPepe** [comment]
cc: @jefferyto and @commodo

File: lang/python/python-tornado/Makefile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+define Package/python3-tornado
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Web framework and asynchronous networking library
+  URL:=https://tornadoweb.org
+  DEPENDS:= \
+    +python3-asyncio \
+    +python3-codecs \
+    +python3-ctypes \
```
Comment (jefferyto) [inline lang/python/python-tornado/Makefile:34]: ctypes appears to only be imported in [tornado/platform/windows.py](https://github.com/tornadoweb/tornado/blob/v6.0.4/tornado/platform/windows.py#L4-L5), which I believe is only used when the OS is Windows, so I don't think `+python3-ctypes` is necessary.

Comment (BKPepe) [reply lang/python/python-tornado/Makefile:34]: 🤦 You are correct.

**Review by jefferyto — APPROVED** [review]
Thanks! (Don't forget to squash the commits before merging :joy:)

**Review by commodo — APPROVED** [review]

**Pull request merged**